client/app/components/Header/HeaderProfile.jsx

```diff
@@ -2,11 +2,10 @@
 import React from 'react';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 import { faBell } from '@fortawesome/free-solid-svg-icons';
-import { Avatar } from '../Avatar';
 import type { Profile } from './index';
```

Comment:
I think still might be the issue. `Profile` comes from [Header/index.jsx](https://github.com/ifmeorg/ifme/pull/1584/files#diff-92470f5ab85c5de61d20bac88b5732deR19) `Header/index.jsx` pulls in `<HeaderProfile />` so that's a circular dependency.

Reply:
Hi Julian,

That's what I am getting at, I'm not sure this can be refactored to resolve the circular import error.
**HeaderProfile**
```
import type { Profile } from './index';

export type Props = {
  profile: Profile,
};
```

**index.jsx**
```
import { HeaderProfile } from './HeaderProfile';

export type Profile = {
  avatar?: string,
  name: string,
  profile: Link,
  account: Link,
  notifications: {
    plural: string,
    none: string,
    clear: string,
  },
};
```
Profile is being imported and exported between the two files and I don't know if the circular logic can be broken?
I can finish up the css refactoring

Reply:
Hi Julia,

Thanks for the response, sorry about misspelling your name!

Reply:
@julianguyen 
Hey, I separated the profile into it's own file profile.js
I then imported it into the two files and ran into the same  import/no-cycle error.
I'm update my PR with this new code for you to review.

Thanks!